mobile/assets/i18n/fr-FR.json

```diff
@@ -268,8 +268,8 @@
   "map_zoom_to_see_photos": "Dézoomer pour voir les photos",
   "monthly_title_text_date_format": "MMMM y",
   "motion_photos_page_title": "Photos avec mouvement",
-  "multiselect_grid_edit_date_time_err_read_only": "Cannot edit date of read only asset(s), skipping",
-  "multiselect_grid_edit_gps_err_read_only": "Cannot edit location of read only asset(s), skipping",
+  "multiselect_grid_edit_date_time_err_read_only": "Impossible de modifier l'emplacement du ou des actifs en lecture seule, la modification est donc ignorée.",
+  "multiselect_grid_edit_gps_err_read_only": "Impossible de modifier l'emplacement du ou des actifs en lecture seule, la modification est donc ignorée.",
```

Comment:
```suggestion
  "multiselect_grid_edit_date_time_err_read_only": "Impossible de modifier l'emplacement de médias en lecture seule, la modification est donc ignorée.",
  "multiselect_grid_edit_gps_err_read_only": "Impossible de modifier l'emplacement de médias en lecture seule, la modification est donc ignorée.",
```

Reply:
I think `actif` is not really a good translation of `asset`. I much prefer to use `media` and there are a lot of references to `asset` in the translation